files/puppet_helper.groovy

```diff
@@ -180,6 +186,108 @@ class Util {
   }
 } // class Util
 
+/**
+ * Helper class to be used when creating a new instance of FileCredentialsImpl
+ */
+public class DirectFileItem implements FileItem {
```

Comment:
[FileItem](https://commons.apache.org/proper/commons-fileupload/apidocs/org/apache/commons/fileupload/FileItem.html) interface is from apache.commons ... I thought usage of [DiskFileItem](https://commons.apache.org/proper/commons-fileupload/apidocs/org/apache/commons/fileupload/disk/DiskFileItem.html) would be overkill since there is only one way how to provide file contents with ```jenkins_credentials```.